Lib/unittest/mock.py

```diff
@@ -847,8 +854,8 @@ def assert_has_calls(self, calls, any_order=False):
         if not any_order:
             if expected not in all_calls:
                 raise AssertionError(
-                    'Calls not found.\nExpected: %r\n'
-                    'Actual: %r' % (_CallList(calls), self.mock_calls)
+                    'Calls not found.\nExpected: %r%s'
+                    % (_CallList(calls), self._call_list_as_string())
```

Comment:
Hum, I would prefer to not replace "Actual" with "Calls". Maybe add an optional parameter to _call_list_as_string(prefix="Calls")?